docs/reference/changelog.rst

```diff
@@ -5,6 +5,11 @@ Changelog
 X.Y.Z (yyyy-mmm-dd)
 -------------------
 
+Application
+===========
+
+- Fix: use CRAFT_PARALLEL_BUILD_COUNT, rather than always building on one CPU.
```

Comment:
You may want to clarify that this is about it not being evaluated when used in a project's metadata file.  It works fine when plugins get it as a class variable.

Reply:
Thanks, updated!